spec/latest/triple-pattern-fragments/index.html

```diff
@@ -229,7 +229,7 @@
         <dt>hypermedia controls</dt>
         <dd>
           A Triple Pattern Fragment MUST contain hypermedia controls
-          that allow retrieving any other Triple Pattern Fragment of the same dataset.
+          that allow retrieval of any other Triple Pattern Fragment of the same dataset.
```

Comment:
Any reasoning why the continuous tense is wrong here? I learned that it makes documents more lively and is thus to be preferred.